roadmap.md

```diff
@@ -23,26 +23,40 @@ implement the spec.
 1. Create and deploy a website that features a simple overview, email list and directs visitors to Github.
 1. Store all website assets in the CloudEvents repository, under the governance
 of the working group.
-
-*0.2*
-
 1. Have at least 2 implementations of the specification that can demonstrate interoperability.
 1. Include a specification for mapping the CloudEvents specification to HTTP.
 1. Include a specification for mapping the CloudEvents specification to JSON.
 1. Changes to the spec to facilitate adoption.
```

Comment:
delete? -- the interop demo was done very quickly after 0.1 was adopted so I wouldn't say this really happened, and 0.2 has "Incorporate learnings and feedback from interop demo to support wider adoption."

Reply:
While the interop demo was completed just after 0.1, I consider it to have been done *for* 0.1 - and I'm not sure the exact ordering really matters when we're just talking about a matter of days.  Besides, saying its done for v0.2 would be even more misleading since by the time 0.2 goes out the 0.1 demo may not even be around - its already pretty close to gone now.